src/DotLiquid.Tests/StandardFilterTests.cs

```diff
@@ -1572,5 +1603,80 @@ public void TestWhere_NonStringCompare()
 {% endfor %}",
                 localVariables: Hash.FromAnonymousObject(new { products }));
         }
+
+        [Test]
+        public void TestConcat()
+        {
+            var array1 = new String[] { "one", "two" };
+            var array2 = new String[] { "alpha", "bravo" };
+            var arrayCombined = new String[] { "one", "two", "alpha", "bravo" };
+
+            Assert.AreEqual(null, StandardFilters.Concat(null, null));
+            Assert.AreEqual(array1, StandardFilters.Concat(array1, null));
+            Assert.AreEqual(array1, StandardFilters.Concat(null, array1));
+            Assert.AreEqual(arrayCombined, StandardFilters.Concat(array1, array2));

```

Comment:
Should IEnumerable return be tested by CollectionAssert.AreEqual?

Reply:
Changed to CollectionAssert.AreEqual.